Sources/ConsoleKit/Command/CommandError.swift

```diff
@@ -5,15 +5,51 @@ public struct CommandError: Error, CustomStringConvertible {
 
     /// See `Debuggable`.
     public let reason: String
+    
+    /// The name of the attempted command
+    private let name: String?
+    
+    /// All available commands
+    private let availableCommands: [String]
```

Comment:
Is there a reason that these properties are marked as `private` instead of being `public`?